### ISSUE TITLE
ScrollStep initializer incorrectly marked with `internal` access control

### ISSUE THREAD

**Issue by cyonix** [cbpowell/MarqueeLabel#192]
### Please follow the general troubleshooting steps first:

- [x] Updated to latest version of MarqueeLabel
- [x] Read the README, in particular the [Special Notes](https://github.com/cbpowell/MarqueeLabel#special-notes)
- [x] For general questions about using MarqueeLabel, post on StackOverflow with the [#MarqueeLabel tag](http://stackoverflow.com/questions/tagged/marqueelabel)

### Usage Details
  - Language: Swift
  - Platform: iPhone/iPad
  

Initializer for `ScrollStep` struct is given by:
```swift
public struct ScrollStep: MarqueeStep {
    ...
    init(timeStep: CGFloat, timingFunction: UIViewAnimationCurve = .linear, position: Position, edgeFades: EdgeFade)
    ...
}
``` 
which makes it impossible for one to create a custom `scrollSequence` on the `MarqueeLabel`:

```swift
open class MarqueeLabel: UILabel, CAAnimationDelegate {
    ...
    open var scrollSequence: Array<MarqueeStep>?
    ...
}
```

Would it be possible to mark that initializer as `public` . ?

**Comment by cbpowell** [comment]
Definitely! Good catch, I'll change it to `public` in release 3.1.1.

(Side note: I tried to `lint` the 3.1.1 release in Cocoapods last night but got a few errors. It might be related to the Xcode beta so I'll have to work that out today before I can push the release)

**Comment by ThuggishNuggets** [comment]
Thanks @cyonix!

**Comment by cbpowell** [comment]
Fixed this in [Release 3.1.1](https://github.com/cbpowell/MarqueeLabel/releases/tag/3.1.1) - thanks @cyonix!
